src/Install/Steps/WriteSettings.php

```diff
@@ -71,7 +71,7 @@ private function getDefaults()
             'theme_dark_mode' => '0',
             'theme_primary_color' => '#4D698E',
             'theme_secondary_color' => '#4D698E',
-            'welcome_message' => 'This is beta software and you should not use it in production.',
+            'welcome_message' => 'Have a look around, and get used to your new forum!',
```

Comment:
I think we should link to Discuss here. 

Enjoy your new forum! Hop over to `https://discuss.flarum.org` if you have any questions, or to join our community!

Idk, not sure how to word this.

Reply:
I like that, I think that's simple enough and nice.

Reply:
But the link won't autolink, no?

Reply:
Ah, perhaps just `discuss.flarum.org` or `flarum.org` ?